packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_2_wrappers/sk2_product_wrapper.dart

```diff
@@ -400,6 +400,17 @@ class SK2Product {
     return result.convertFromPigeon();
   }
 
+  /// Checks if the user is eligible for a introductory offer.
+  static Future<bool> isIntroductoryOfferEligible(
```

Comment:
uber nit: since this is still a public API, consider adding the product must be a subscription too?